menstoreproject/catalog/models.py

```diff
@@ -24,6 +25,9 @@ class Product(models.Model):
     material = models.CharField(max_length=50, verbose_name='Матеріал')
     quantity = models.IntegerField()
 
+    def get_absolute_url(self):
+        return reverse('view_product', kwargs={'pk': self.pk})
```

Comment:
kwargs={'pk': self.pk} better to use slug. slug is developed to use it as identifier in URLs.